src/collider/README.md

```diff
@@ -38,3 +37,49 @@ A websocket-based signaling server in Go.
 
     go test collider
 
+## Deployment
+
+Change [roomSrv](https://github.com/webrtc/apprtc/blob/master/src/collider/collidermain/main.go#L16) to your AppRTC server instance e.g.
+
+```go
+var roomSrv = flag.String("room-server", "https://your.apprtc.server", "The origin of the room server")
+```
+
+Then repeat step 6 in the Building section and then start it on your server by following the Running instructions.
+
+### Certificates
+If you are deploying this in production, you should use certificates so that you can use secure websockets. Place the `cert.pem` and `key.pem` files in `/cert/`. E.g. `/cert/cert.pem` and `/cert/key.pem`
+
+### Auto restart
+1\. Add a `/collider/start.sh` file:
```

Comment:
That's how the original Collider VM's are setup, we can ofc change it but I do not see the point of having different instructions. These are single purpose machines, putting all the folders in root makes them easy to find, sure you could argue they could all be put in the home folder as well...